package/gluon-ssid-changer/luasrc/lib/gluon/config-mode/model/admin/ssid-changer.lua

```diff
@@ -0,0 +1,41 @@
+local uci = require('simple-uci').cursor()
+local util = require 'gluon.util'
+
+local pkg_i18n = i18n 'gluon-ssid-changer'
+
+local f = Form(pkg_i18n.translate('Offline-SSID'))
+
+local s = f:section(Section, nil, pkg_i18n.translate(
+	'Here you can enable to automatically change the SSID to the Offline-SSID '
+  .. 'when the node has no connection to the selected Gateway.'
+))
+
+local enabled = s:option(Flag, 'enabled', pkg_i18n.translate('Enabled'))
+enabled.default = uci:get_bool('ssid-changer', 'settings', 'enabled')
+
+-- local prefix = s:option(Value, 'prefix', pkg_i18n.translate('First part of the Offline SSID'))
```

Comment:
many commented out lines in this file?

Reply:
Comments are deleted by gluonSourceDiet.

I wasnt sure, if we need those lines here again maybe?

Reply:
having a comment with information is not the same as having code that isn't used.
i can't say if you need this again, but if you do you should uncomment it ;)

Reply:
I removed the code for now. If we want to make those values configurable later too, we could add the code from git history in the Freifunk Nord branch